cmd/gctcli/futures.go

```diff
@@ -298,6 +217,268 @@ var futuresCommands = &cli.Command{
 				},
 			},
 		},
+		{
+			Name:      "getcollateralmode",
+			Aliases:   []string{"gcm"},
+			Usage:     "gets the collateral mode for an exchange asset",
+			ArgsUsage: "<exchange> <asset>",
+			Action:    getCollateralMode,
+			Flags: []cli.Flag{
+				&cli.StringFlag{
+					Name:    "exchange",
+					Aliases: []string{"e"},
+					Usage:   "the exchange to retrieve futures positions from",
+				},
+				&cli.StringFlag{
+					Name:    "asset",
+					Aliases: []string{"a"},
+					Usage:   "the asset type of the currency pair, must be a futures type",
+				},
+			},
+		},
+		{
+			Name:      "setcollateralmode",
+			Aliases:   []string{"scm"},
+			Usage:     "sets the collateral mode for an exchange asset",
+			ArgsUsage: "<exchange> <asset> <collateralmode>",
+			Action:    setCollateralMode,
+			Flags: []cli.Flag{
+				&cli.StringFlag{
+					Name:    "exchange",
+					Aliases: []string{"e"},
+					Usage:   "the exchange to retrieve futures positions from",
+				},
+				&cli.StringFlag{
+					Name:    "asset",
+					Aliases: []string{"a"},
+					Usage:   "the asset type of the currency pair, must be a futures type",
+				},
+
+				&cli.StringFlag{
+					Name:    "collateralmode",
+					Aliases: []string{"collateral", "cm", "c"},
+					Usage:   "the collateral mode type, such as 'single', 'multi' or 'global'",
+				},
+			},
+		},
+		{
+			Name:      "setleverage",
```

Comment:
```
go run ./... futures setleverage --exchange=okx --asset=futures -pair=BTC-USDC-230929 -m=isolated -yolo=1
2023/08/14 11:27:42 rpc error: code = Unknown desc = Okx unsuccessful HTTP status code: 400 raw response: {"code":"51000","data":[],"msg":"Parameter posSide error"}, authenticated request failed
```

Using v5 keys

Cant test binance

Reply:
Thank you for this pickup! Was a little bit important